src/commander/flags.cr

```diff
@@ -11,7 +11,15 @@ class Commander::Flags
     flags.each { |flag| yield flag }
   end
 
-  def add(flag)
+  def add(*multi)
+    add multi.to_a
+  end
+
+  def add(multi : Array(Commander::Flag))
```

Comment:
I figured. ;) 

But I'm fine with using `flags` over `multi` since the scope is so small it won't be confusing.